modules/ROOT/pages/view.adoc

```diff
@@ -13,20 +13,19 @@ Views display your saved filter selection, including:
 * New services deployed that match the selected environments and tags
 * If the *Show dependents and dependencies* option is selected and the a service is deployed and related to a selected service
 
-Views that you have access to appear in a list in the *View* card (click the drop-down arrow to see the list). This gives you easy access to all the views that are available to you, and saves you time because you don't need to readjust filters every time you access Visualizer to get the view that is relevant to you.
+Views that you have access to appear in a list in the *View* card (click the drop-down arrow to see the list). This gives you easy access to all the views that are available to you, and saves you time because you don't need to readjust filters every time you access Anypoint Visualizer to get the view that is relevant to you.
```

Comment:
```suggestion
For quick access and to save you from readjusting the filters every time you use Anypoint Visualizer, all the views to which you have access appear in a list in the *View* card (click the drop-down arrow to see the list). 
```

Reply:
Written as it was to emphasize the UI object before anything else.